firmware/application/apps/ui_freqman.cpp

```diff
@@ -284,6 +303,15 @@ void FrequencyManagerView::on_del_entry() {
             }
         });
 }
+bool FrequencyManagerView::forbid_delete_system_item_helper(NavigationView& nav) {
+    // this is just a modal, however, it's been forbidden in those handler.
+
+    if (current_is_system_item) {
+        nav.display_modal("Forbid", "Can't do that to \nsystem item.\nIf you have to,\ndo it with file manager.");
```

Comment:
This error message isn't really correct if the user is trying to edit a line in the file; I think it should say "Notepad" instead of "File Manager" in that case.